src/posix/platform/platform-posix.h

```diff
@@ -245,11 +239,38 @@ void platformUartProcess(const fd_set *aReadFdSet, const fd_set *aWriteFdSet, co
 /**
  * This function initializes platform netif.
  *
- * @param[in]   aInstance       A pointer to the OpenThread instance.
+ * @note This function is called before OpenThread instance is created.
+ *
  * @param[in]   aInterfaceName  A pointer to Thread network interface name.
  *
  */
-void platformNetifInit(otInstance *aInstance, const char *aInterfaceName);
+void platformNetifInit(const char *aInterfaceName);
+
+/**
+ * This function sets up platform netif.
```

Comment:
nit: To be consistent with comments below,
 `This function sets up platform network interface.`

Reply:
The document follows `platformNetifInit` which also used the wording `platform netif`.

Reply:
fixed